### PR TITLE
Unittest slack notifier

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Description
Separated unit test slack messages to a different message

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@yaakovi This could only be merged after you release the SDK update

**Comment by ShahafBenYakir** [comment]
@yaakovi can you approve this? Now that the new SDK version is released we can merge this to get the unit test changes.

File: Tests/scripts/download_demisto_conf.sh [hunk 1]
```diff
@@ -45,9 +45,11 @@ if [ "$?" != "0" ]; then
...
     rm -rf ./test_configuration.zip
+    echo "ERROR: Found a branch with the same name in contest-test-conf conf.json - $UNDERSCORE_CIRCLE_BRANCH.\n Merge it in order to merge the current branch into content repo."
+    exit 1
+
 fi
 
 set -e
-echo "using instance:"
-cat instance.json
+rm instance.json
```
Comment (yaakovi) [inline Tests/scripts/download_demisto_conf.sh]: remove this line

File: Tests/scripts/slack_notifier.py [hunk 2]
```diff
@@ -149,18 +170,21 @@ def get_fields(build_number):
...
+def slack_notifier(build_url, slack_token, env_results_file_name, container):
     branches = run_command("git branch")
     branch_name_reg = re.search(r'\* (.*)', branches)
     branch_name = branch_name_reg.group(1)
 
     if branch_name == 'master':
         print_color("Starting Slack notifications about nightly build", LOG_COLORS.GREEN)
         print("Extracting build status")
-        content_team_attachments, _ = get_attachments(build_url, env_results_file_name)
+        if container:
```
Comment (yaakovi) [inline Tests/scripts/slack_notifier.py]: add comments
container 0: test playbooks
container 1: unit tests

File: Tests/scripts/download_demisto_conf.sh (hunk 1 above)
Comment (ShahafBenYakir) [reply Tests/scripts/download_demisto_conf.sh]: Why? We said we don't want it

Comment (yaakovi) [reply Tests/scripts/download_demisto_conf.sh]: we are not using the instance so the print message shouldn't be there.
but no need to delete it.
instead, you can remove lines 39-45

**Pull request merged**